src/mem/ruby/protocol/GPU_VIPER-TCC.sm

```diff
@@ -105,16 +105,17 @@ machine(MachineType:TCC, "TCC Cache")
   }
 
   structure(TBE, desc="...") {
-    State TBEState,     desc="Transient state";
-    DataBlock DataBlk,  desc="data for the block";
-    bool Dirty,         desc="Is the data dirty?";
-    bool Shared,        desc="Victim hit by shared probe";
-    MachineID From,     desc="Waiting for writeback from...";
-    NetDest Destination, desc="Data destination";
-    int numAtomics,     desc="number remaining atomics";
-    int atomicDoneCnt,  desc="number AtomicDones triggered";
-    bool isGLCSet,      desc="Bypass L1 Cache";
-    bool isSLCSet,      desc="Bypass L1 and L2 Cache";
+    State TBEState,             desc="Transient state";
```

Comment:
Why are all of these changed?  Did you mess around with spacing or something?  Seems like only 118 should be changed ...

Reply:
The longer name of the added entry pushed back the indentation of the description, so I further indented the rest to match.

Reply:
I see, thanks -- I guess this is ok